pandera/decorators.py

```diff
@@ -154,6 +154,26 @@ def _wrapper(
                     check=e.check,
                     check_index=e.check_index,
                 )
+        elif obj_getter is None and kwargs:
+            # get the first key in the same order specified in the
+            # function argument.
+            args_names = _get_fn_argnames(fn)
+
+            try:
+                kwargs[args_names[0]] = schema.validate(
+                    kwargs[args_names[0]], *validate_args
+                )
+            except errors.SchemaError as e:
+                msg = (
+                    "error in check_input decorator of function '%s': %s" %
+                    (fn.__name__, e)
+                )
+                raise errors.SchemaError(
+                    schema, args_names[0], msg,
```

Comment:
this should be the arg value, so `kwargs[args_names[0]]`

Reply:
good catch, thank you